SIT.Manager/ViewModels/Play/CharacterSummaryViewModel.cs

```diff
@@ -52,6 +68,20 @@ public CharacterSummaryViewModel(AkiServer server, AkiMiniProfile profile, ILoca
             NextLevel = Profile.MaxLevel;
         }
 
+        Task.Run(async () =>
+        {
+            if(!string.IsNullOrEmpty(profile.Side) && !profile.Side.Equals("unknown", StringComparison.InvariantCultureIgnoreCase))
+            {
+                string cacheKey = $"side_{Profile.Side} icon";
+                CacheValue<Bitmap> cacheVal = await _cachingService.InMemory.GetOrComputeAsync<Bitmap>(cacheKey, async (key) =>
+                {
+                    return new(await _akiServerRequestingService.GetAkiServerImage(server, $"launcher/side_{profile.Side.ToLower()}.png"));
+                });
+                if(cacheVal.HasValue && cacheVal.Value != null)
+                    SideImage = cacheVal.Value;
+            }
```

Comment:
Would this make more sense to be a function that you just await in the Task run?